includes/Core/Permissions/Permissions.php

```diff
@@ -108,34 +107,36 @@ public function __construct( Context $context, Authentication $authentication =
 			self::AUTHENTICATE        => 'manage_options',
 
 			// Allow contributors and up to view their own post's insights.
-			self::VIEW_POSTS_INSIGHTS => 'edit_posts',
+			// TODO change to map to edit_posts when Site Kit supports non admin access.
+			self::VIEW_POSTS_INSIGHTS => 'manage_options',
 
 			// Allow editors and up to view the dashboard and module details.
-			self::VIEW_DASHBOARD      => 'edit_others_posts',
-			self::VIEW_MODULE_DETAILS => 'edit_others_posts',
+			// TODO change to map to edit_others_posts when Site Kit supports non admin access.
+			self::VIEW_DASHBOARD      => 'manage_options',
+			self::VIEW_MODULE_DETAILS => 'manage_options',
 
 			// Allow administrators and up to manage options and set up the plugin.
 			self::MANAGE_OPTIONS      => 'manage_options',
 			self::SETUP               => 'manage_options',
-
-			// Allow to differentiate between authors and contributors.
-			self::PUBLISH_POSTS       => 'publish_posts',
 		);
 
 		$this->meta_to_core = array(
 			// Allow users that can edit a post to view that post's insights.
-			self::VIEW_POST_INSIGHTS => 'edit_post',
+			// TODO change to map to edit_post when Site Kit supports non admin access.
+			self::VIEW_POST_INSIGHTS => 'manage_options',
```

Comment:
@Hazlitte I don't see that IB mentions `VIEW_POST_INSIGHTS` to be updated. Are you sure it needs to be updated too?

Reply:
@eugene-manuilov  No you are right it doesn't I will correct this.